mobile/src/main/java/ch/liip/timeforcoffee/helper/ConnectionsDeserializer.java

```diff
@@ -0,0 +1,60 @@
+package ch.liip.timeforcoffee.helper;
```

Comment:
This file should maybe be moved to the "api". Because it is the project that deserialises/maps the data from the api

Reply:
Yeah. Forgot to do it. Good catch.